### DISCUSSION TITLE
Page.overlay.append NOT show IMAGE

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#2436] [Q&A]
### Question

this bug or not . but i test show image not show in overlay

### Code sample

```python
from flet import *
from math import pi
def main(page: Page):
    page.window_width = 400

    def change_size(e):
        img_view.scale = e.control.value
        page.update()

    def change_rotate(e):
        img_view.rotate.angle += pi /2 
        page.update()

    img_view = Container(
        animate=animation.Animation(200,"easeIn"),
        content=Image(src="assets/burger.jpg"),
        alignment=alignment.center,
        rotate=transform.Rotate(0, alignment=alignment.center),
        )

    con_controls = Container(
        content=Column([
            Slider(min=1,max=2,on_change=change_size),
            ElevatedButton("rotate",on_click=change_rotate)
            ])
        )

    def close_view(e):
        page.overlay.clear()
        page.update()

    def openview(e):
        page.overlay.append(
            Container(
                bgcolor="black",
                on_click=close_view,
                content=Column([
                img_view,
                con_controls
            ])
                )
            )
        page.update()

    page.add(
        Container(
            content=img_view,
            ink=True,
            on_click=openview
            )
        )

app(main, view=AppView.WEB_BROWSER, assets_dir="assets")
```

### Error message

```bash
not work
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by burhansvural** [comment]
![burger](https://github.com/flet-dev/flet/assets/149883930/69c3c217-741d-43fd-91da-42e6ee7dab79)

<code>
from flet import *
from math import pi
def main(page: Page):
    page.window_width = 400

    def change_size(e):
        img_view.scale = e.control.value
        page.update()

    def change_rotate(e):
        img_view.rotate.angle += pi /2 
        page.update()

    img_view = Container(
        animate=animation.Animation(200,"easeIn"),
        content=Image(src="assets/burger.jpg",
                      width=500,
                      height=500),
        alignment=alignment.center,
        rotate=transform.Rotate(0, alignment=alignment.center),
        )

    con_controls = Container(
        content=Column([
            Slider(min=1,max=2,on_change=change_size),
            ElevatedButton("rotate",on_click=change_rotate)
            ])
        )

    def close_view(e):
        page.overlay.clear()
        page.update()

    def openview(e):
        page.add(
            Container(
                bgcolor="black",
                on_click=close_view,
                content=Column([
                img_view,
                con_controls
            ])
                )
            )
        page.update()

    page.add(
        Container(
            content=img_view,
            ink=True,
            on_click=openview
            )
        )

app(main, view=AppView.WEB_BROWSER, assets_dir="assets")
</code>
